src/List.tsx

```diff
@@ -0,0 +1,34 @@
+function List() {
+  return (
+    <div>
+      <ul>
+        <li>A</li>
+        <li>B</li>
+        <ol type="1">
```

Comment:
It looks like you're setting the type attribute to 1 for the <ol> element. This is the default value, so you don't need to specify it. Removing it will make the code cleaner.

Reply:
Type="1" has been resolved and removed.